scripts/gen-gd2conf.sh

```diff
@@ -19,4 +20,5 @@ cat >$OUTPUT <<EOF
 localstatedir = "$GD2STATEDIR"
 logdir = "$GD2LOGDIR"
 logfile = "$GD2.log"
+rundir = "$RUNDIR"
```

Comment:
The rundir needs to be GD2 specific, ie. `$RUNDIR/$GD2`. Better to setup $GD2RUNDIR and use it here.

Reply:
Done